static_src/components/app_page.jsx

```diff
@@ -82,10 +82,24 @@ export default class AppPage extends React.Component {
                 </tr>
               </tbody>
             </table>
+            <aside>
+              <p>To start or stop an app, view the <a
+                href="https://docs.cloudfoundry.org/devguide/deploy-apps/deploy-app.html"
+                target="_blank">
+                <span>&nbsp;</span>deployment guide</a> for more information.
```

Comment:
It's to ensure spacing in React. If you move to a new line, if won't always but a space.